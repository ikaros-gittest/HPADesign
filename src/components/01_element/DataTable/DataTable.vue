<template>
    <v-data-table 
        :headers="headers"
        :items="items"
    />
</template>
<script lang="ts">
import Vue from 'vue';


export default Vue.extend({
    props: {
        //headers: Array,
        //items: Array,
    },
    data () {
        return {
            headers: [
                {
                    text: 'key',
                    value: 'key'
                },
                {
                    text: 'item',
                    value: 'item'
                }
            ],
            items: [
                { key: "key1", item: "item1" },
                { key: "key2", item: "item2" },
            ],
        }
    }
})
</script>