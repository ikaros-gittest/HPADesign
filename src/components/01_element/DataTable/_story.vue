<template>
	<DataTable
			:headers="headers"
			:items="items"
	/>
</template>
<script lang="ts">
import Vue from 'vue';
import { storiesOf } from '@storybook/vue';

import DataTable from './DataTable.vue';

const testStory = Vue.extend({
	components: { DataTable },
	data () {
		return {
			headers: [
					{
							text: 'key',
							value: 'key'
					},
					{
							text: 'item',
							value: 'item'
					}
			],
			items: [
					{ key: "key1", item: "item1" },
					{ key: "key2", item: "item2" },
			],
		}
	}
});
export default testStory;

storiesOf("DataTable", module)
    .add("DataTable", () => testStory);

</script>