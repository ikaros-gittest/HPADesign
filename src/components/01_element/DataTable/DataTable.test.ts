import { mount } from '@vue/test-utils';
import DataTable from './DataTable.vue';

describe('DataTable', () => {
    it('snapshot test', () => {
        const wrapper = mount(DataTable);
        expect(wrapper.element).toMatchSnapshot();
    });
});