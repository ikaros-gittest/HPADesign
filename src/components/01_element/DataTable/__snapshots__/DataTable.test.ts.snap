// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable snapshot test 1`] = `<v-data-table />`;
